sdk/communication/communication-chat/review/communication-chat.api.md

```diff
@@ -37,19 +37,18 @@ export interface AddParticipantsRequest {
     participants: ChatParticipant[];
 }
 
-// @public
-export type AttachmentType = "teamsInlineImage" | "teamsImage" | "file";
-
 // @public
 export interface ChatAttachment {
-    attachmentType: AttachmentType;
-    contentType?: string;
+    attachmentType: ChatAttachmentType;
     id: string;
     name?: string;
     previewUrl?: string;
-    url: string;
+    url?: string;
 }
 
+// @public
+export type ChatAttachmentType = "image" | "file" | "unknown";
```

Comment:
can you mention these changes in the changelog? Usually that is the starting point when folks ask what is different in a new package version

Reply:
thanks Jeff, change log updated